code/modules/events/vent_clog.dm

```diff
@@ -47,7 +47,21 @@
 		"hair_dye",
 		"sugar",
 		"white_glitter",
-		"growthserum"
+		"growthserum",
+
+		"cooking_oil",
+		"frostoil",
+		"sodiumchloride",
+		"cornoil",
+		"uranium",

```

Comment:
This will make rad zones that are annoying to clean and can be meme'ed into EMPs fields when water stashed on them